libs/login/WordPressLoginFlow/src/main/res/values-v23/dimens.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+
+    <dimen name="status_bar_height_offset">24dp</dimen>
```

Comment:
Since we're assuming the status bar is always going to have a `24dp` height, I believe this is causing the issue @mattmiklic mentioned with the Pixel 3XL notch:

<img width="400" alt="Captura de Tela 2020-10-16 às 19 54 55" src="https://user-images.githubusercontent.com/830056/96322455-86512f80-0fef-11eb-8c64-713ca7db057c.png">

Another thing, if I'm not mistaken this is supposed to be used with the new `Theme.LoginFlow.TransparentStatusBar`, right? If so, since this change is being made within the Login Library, we also need to preserve the behavior for when this theme is not being used, otherwise it might cause some unexpected results in the Woo app, for example.